app.js

```diff
@@ -178,6 +178,27 @@ client.once("ready", async () => {
 
     await channel.send({ content: "**Leetcode Weekly**", embeds: [weeklyProblemMessage] })
 
+    // Get Stats
+    // Server Members
+    //console.log(client.guilds.cache);
+    let dominationGuild = client.guilds.resolve(config.GUILD_ID);
+    console.log(dominationGuild);
+    //console.log(dominationGuild.members.cache);
+    let members = await dominationGuild.members.fetch();
```

Comment:
This is failing because we have some permissions issues, we need to figure this out before proceeding